shop/views.py

```diff
@@ -1,10 +1,21 @@
 from django.shortcuts import render
+from .models import Item
 # Create your views here.
 def home(request):
     return render(request, 'shop/index.html')
 
 def dashboard(request):
-    return render(request, 'shop/dashboard.html')
+    items = Item.objects.order_by('name')
```

Comment:
All items should not be rendered to the page. Only the items specific to the Seller must be rendered.